app.R

```diff
@@ -0,0 +1,843 @@
+                        "China-Males" = "China-Males",
+                        "United Kingdom-Females" = "UK-Females",
+                        "United Kingdom-Males" = "UK-Males",
+                        "United States-Females" = "US-Females",
+                        "United States-Males" = "US-Males"
+                      ),
+                      selected = "China-Females"
+                    ),
+                    radioButtons(
+                      inputId = "cohortRight",
+                      label = "Right side of the cohort pyramid",
+                      choices = c(
+                        "China-Females" = "China-Females",
+                        "China-Males" = "China-Males",
+                        "United Kingdom-Females" = "UK-Females",
+                        "United Kingdom-Males" = "UK-Males",
+                        "United States-Females" = "US-Females",
+                        "United States-Males" = "US-Males"
+                      ),
+                      selected = "China-Males"
+                    )
+                  )
+                ),
+                column(
+                  width = 8,
+                  offset = 0,
+                  plotOutput("pyramidCountrySex") %>%
+                    withSpinner(color = boastPalette[3])
+                )
+              )
+            )
+          )
+        ),
+        #### Time Population Pyramids ----
+        tabItem(
+          tabName = "timePry",
+          ### ISSUE move into a Single Page
+          h2("Time and Population Pyramids"),
+          p("Connecting population pyramids with time allows us to explore changes
+            in a country's population. This allows us to see when there might be
+            large increases in the population as well as when a country might
+            suffer from catesphroic loss of life."),
+          p("Explore both population pyramids for the United Kingdom and the
+            United States. What do you notice? What might explain any big changes?"),
+          fluidRow(
+            column(
+              width = 4,
+              offset = 0,
+              wellPanel(
+                selectInput(
+                  inputId = "actualCountry",
+                  label = "Select a country",
+                  choices = c(
+                    "United Kingdom",
+                    "United States"
+                  ),
+                  selected = "United States"
+                ),
+                sliderInput(
+                  inputId = "yearInterval",
+                  label = "Year",
+                  min = 1900,
+                  max = 2018,
+                  value = 1900,
+                  round = TRUE,
+                  sep = "",
+                  animate = animationOptions(interval = 2000, loop = FALSE)
+                )
+              )
+            ),
+            column(
+              width = 8,
+              offset = 0,
+              plotOutput("timePyramid")
```

Comment:
I also really like the added feature of getting to press the pay button and the graph automatically increases by year. Is there any way to make it go faster or allow the student to choose a speed?

Reply:
~No, there isn't a way for the user to set the speed.~ Well, there might be a hack job way to make some user control but I'm not sure how viable that would be. 

Currently the animation is set to pause 2000 milliseconds between ticks. Is this too long? We run into an issue where if the wait time is too short the slider will get ahead of the plot by multiple ticks.

Reply:
I see! The speed isn't a big issue at all the animation still looks great!